packages/flutter/test/widgets/selection_test.dart

```diff
@@ -0,0 +1,7 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+void main() {
+  // TODO(chunhtai): add tests for selection container.
```

Comment:
this should happen before we land this, or we shouldn't land the file. no point landing the file with just the todo :-)

Reply:
yes, I am in the progress of adding more tests, please ignore the test file for now.  I would like to get some feedback on code structure first in the meanwhile in case I may need to rewrite a bunch of tests if API changes as the result of addressing feedbacks.